app-android/calendar/build.gradle.kts

```diff
@@ -89,7 +89,7 @@ android {
 		val variant = this
 		variant.outputs.configureEach {
 			val flavor = variant.productFlavors[0].name
-			this.outputFile.renameTo(File(this.outputFile.toPath().parent.toString() + "tutanota-$flavor-${variant.buildType.name}-${variant.versionName}.apk"))
+			(this as com.android.build.gradle.internal.api.BaseVariantOutputImpl).outputFileName = "tutanota-calendar-$flavor-${variant.buildType.name}-${variant.versionName}.apk"
```

Comment:
`build.gradle.internal.api` sounds like a bad idea 😬 there should be a "proper" way to do it. if there isn't we should leave a comment